nglview/show.py

```diff
@@ -5,6 +5,10 @@
 except ImportError:
     from io import StringIO
 
+from StringIO import StringIO as StringIOOld
```

Comment:
If you want to use the old StringIO, it should be inside the `show_biopython`

And just using `from StringIO import StringIO`